x-pack/plugins/security_solution/public/timelines/components/timeline/search_or_filter/index.tsx

```diff
@@ -30,6 +41,9 @@ interface OwnProps {
 
 type Props = OwnProps & PropsFromRedux;
 
+export const isDataView = (obj: unknown): obj is DataView =>
```

Comment:
I'm surprised dataviews don't come with an api for this. I wonder if you can do `object instanceof DataView` since `DataView` is a "class" iirc.

Reply:
Agree, what else can it be? The `indexPattern` we get from `useSourcererDataView` and check here is typed with `SecuritySolutionDataViewBase`, is that type wrong?